Add tr.org.yongdohyun.ProjTLauncher

YongDo-Hyun opened the pull request:
<!-- ⚠️⚠️  Submission pull request MUST be made against the `new-pr` **base branch** ⚠️⚠️  -->

<!-- 💡 Please go to the preview tab to view the markdown below 💡 -->

### Please confirm your submission meets all the criteria

<!-- 💡 Please replace each `[ ]` with `[X]` when the step is complete 💡 -->

<!-- 💡 Please tick and write 'N/A' with a reason if a checklist item below is not applicable 💡 -->

- [X] Please describe the application briefly. < ProjT Launcher is a community-driven Minecraft launcher based on Prism Launcher, with additional improvements, performance fixes, and usability enhancements. It supports modded and vanilla Minecraft, multiple instances, Java management, and modern UI features. >
- [x] Please attach a video showcasing the application on Linux using the Flatpak. < https://youtu.be/SsbZY1ViUUg >
- [X] The Flatpak ID follows all the rules listed in the [Application ID requirements][appid].
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2] and I agree to them.
- [X] I am the author to the project.
      If not, I contacted upstream developers about this submission. **Link:**

<!-- 💡 Please mention below the GitHub usernames of any additional maintainers needed (if any) 💡 -->

<!-- ⚠️⚠️  Please DO NOT modify anything below this line ⚠️⚠️  -->

[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission

DUB1401 commented:
Is this rebranding clone of [Prism Launcher](https://flathub.org/en/apps/org.prismlauncher.PrismLauncher)? Same screenshots.

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.metainfo.xml:
This file should be (submitted and managed) upstream.

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.desktop:
This file should be (submitted and managed) upstream.

YongDo-Hyun replied:
resolved

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,151 @@
+id: tr.org.yongdohyun.ProjTLauncher
+runtime: org.kde.Platform
+runtime-version: '6.8'
```
current is 6.10

YongDo-Hyun replied:
resolved

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,151 @@
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/libgamemodeauto.a
```
```suggestion
      - '*.a'
```

YongDo-Hyun commented:
> Is this rebranding clone of [Prism Launcher](https://flathub.org/en/apps/org.prismlauncher.PrismLauncher)? Same screenshots.

ProjT Launcher is a fork, but not a rebranding clone.
The project adds new features, performance improvements, patches, UI changes, and upstream contributions. It is an actively developed independent project.

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented:
> we need an answer on why the screenshots on the website are of prism launcher's which is already on flathub

The screenshots used on the website were temporary placeholders taken from Prism Launcher during early development, because ProjT Launcher did not yet have its own UI assets at that time.

ProjT Launcher is now visually different and uses its own UI, branding, and modified features.
We have already prepared our own screenshots and will replace the temporary ones on the website.

This is not a simple rebrand — ProjT Launcher contains:
	•	additional performance patches
	•	new update logic
	•	modified Java management
	•	enhanced multi-instance handling
	•	custom UI theming
	•	our own branding, icons, website, domains, and distribution process

The incorrect screenshots will be updated immediately. Thank you for pointing it out.

hfiguiere reviewed (changes requested):
many things unresolved. it's rude to ask a review without having them resolved.

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented:
> Is this rebranding clone of [Prism Launcher](https://flathub.org/en/apps/org.prismlauncher.PrismLauncher)? Same screenshots.

Replaced screenshots

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,152 @@
+      - /mesa-demos
+      - /share
+
+  - name: enhance
```
this belong to the module that has the main app. Which for some reason is not the last in the list (it should be)

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,152 @@
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        commit: 7b6aead9fb88b3623e3b3725ebb42670cbe4c579 # 3.4
```
```suggestion
        tag: '3.4'
        commit: 7b6aead9fb88b3623e3b3725ebb42670cbe4c579 # 3.4
```

bbhtt commented:
> This is not a simple rebrand — ProjT Launcher contains:

I need the technical details of how these patching etc. was done. Link to PRs or commits will be useful.

bbhtt commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,145 @@
+    cleanup:
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
```
not resolved

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented:
> if i have to parse a commit log like this, this might as well stay blocked
> 
> you either don't know how to write commit messages or are obfuscating
> <img alt="image" width="1241" height="819" src="https://private-user-images.githubusercontent.com/62639087/517840881-59b040b2-5c7a-483a-bb63-6b64fbecf2a4.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjM5MDI3MzUsIm5iZiI6MTc2MzkwMjQzNSwicGF0aCI6Ii82MjYzOTA4Ny81MTc4NDA4ODEtNTliMDQwYjItNWM3YS00ODNhLWJiNjMtNmI2NGZiZWNmMmE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTIzVDEyNTM1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBiZjY4ZTljZmZiYTkxOGFmODhiYjcyYjJjOTY3YThjYzA0NjQwOWNjZGY4MTEyM2UyM2IxMTViNjY2ZTZjNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.bAAFt0KT4JA9dvTHjQMKG4wLOCplgXy8pmcRLZgdyRI">

Hello, the commit messages are numbered (e.g., "144. komit" "komit" means commit in turkish) because this is a development branch with many incremental changes. Here's a summary of the key updates:

- Commits 88-144: Various code updates, including asynchronous backup system implementation, UI improvements, and bug fixes.
- Recent commits: Added prime-run and projtlauncher scripts for Flatpak build, updated manifest per review feedback (removed glu/glxinfo modules, moved main app to end of modules list, added tag to GLFW).
- Build tested successfully with flatpak-builder.

The app is ready for review. Let me know if you need more details on specific commits.

bot, build

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented on tr.org.yongdohyun.ProjTLauncher.metainfo.xml:
```diff
@@ -0,0 +1,78 @@
+  <id>tr.org.yongdohyun.ProjTLauncher</id>
+  <name>ProjT Launcher</name>
+  <summary>Custom Minecraft Launcher to easily manage multiple Minecraft installations at once</summary>
+  <developer id="tr.org.yongdohyun.projtlauncher">
```
resolved

YongDo-Hyun commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,145 @@
+    # Mod drag&drop
+  - --filesystem=xdg-download:ro
+    # FTBApp import
+  - --filesystem=~/.ftba:ro
```
reolved and the downloads folder is needed to manually download and install some mods

YongDo-Hyun commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -0,0 +1,151 @@
+
+modules:
+  # Might be needed by some Controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
```
resolved

YongDo-Hyun commented:
bot, build

YongDo-Hyun commented:
bot, build

hfiguiere commented on prime-run:
where is this ever used? It's not the `command`

YongDo-Hyun commented on prime-run:
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+export __NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia
+exec "$@"
```
Youre right prime offloading shouldnt be built into the application Ive removed the prime-run wrapper users with hybrid gpus can enable nvidia offloading on their system side and flatpaks nvidia plugin already handles this properly thanks for pointing that out

YongDo-Hyun commented:
bot, build

hfiguiere commented on tr.org.yongdohyun.ProjTLauncher.yml:
```diff
@@ -107,6 +107,6 @@ modules:
         url: https://github.com/Project-Tick/ProjT-Launcher.git
         tag: 0.0.3
-        commit: cf1d94b54808841033c9e5ffc20b4771975fae05 # 0.0.3
+        commit: 087a1457c3e254b932105a65bf30008867837e80 # 0.0.3
```
Retagging is bad.

YongDo-Hyun commented:
bot, build
